sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_service_groundedness/_service_groundedness.py

```diff
@@ -0,0 +1,148 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from typing import Optional, Dict
+from typing_extensions import override
+
+from azure.ai.evaluation._common.constants import EvaluationMetrics
+from azure.ai.evaluation._evaluators._common import RaiServiceEvaluatorBase
+
+
+class GroundednessProEvaluator(RaiServiceEvaluatorBase):
```

Comment:
Just to confirm - GroundednessProEvaluator is going be GAed directly?

Reply:
This should be marked as experimental like content safety evaluators

Reply:
No, adding experimental